server/apiCalls/koppsCourseData.js

```diff
@@ -12,9 +12,39 @@ const koppsApi = new BasicAPI({
   defaultTimeout: 10000, // config.koppsApi.defaultTimeout
 })
 
+async function _getApplicationCodeFromLadokUId(ladokuid) {
+  try {
+    const res = await koppsApi.getAsync(`courses/offerings/roundnumber?ladokuid=${ladokuid}`)
```

Comment:
Destructure body direct in await `const {body} = await`

Reply:
done